jenkins/slave.sls

```diff
@@ -0,0 +1,60 @@
+{% from "jenkins/map.jinja" import jenkins, deploy with context %}
+
+include:
+  - .sudoersd
+
+jenkins_deps:
```

Comment:
This would actually cause a clash in salt with an error about "conflicting IDs" if we tried to include both the server and the slave states at the same time, but maybe that is okay as it doesn't make sense for a box to be a jenkins master and a slave at the same time.